pyproject.toml

```diff
@@ -74,7 +74,7 @@ all = [
     ]
 # Needed for readthedocs and man page build. Not being packaged in rpm.
 docs = [
-    "sphinx_rtd_theme>=1.2.2", # The theme pulls a specific sphinx version
+    "renku-sphinx-theme==0.3.0",
```

Comment:
So, I'd suppose that adding `renku-sphinx-theme` here will automatically install the package when `hatch run docs:html` is called. But it seems it's [ignored](https://readthedocs.org/api/v2/build/22485602.txt). @martinhoyer, any idea why not?

Reply:
Installing it with `pip` in the same moment as `hatch` will not help, as that goes into the system or user packages, but `make docs` would use a separate virtualenv constructed by `hatch`.

That said, can you try commenting out `detached=true` in `pyproject.toml`? It was messing with my documentation experiments, using wrong sources. Maybe we could blame it for this as well :)

Reply:
Indeed, `detached=true` should go away.

Reply:
Thanks for the hint! Seems to be working fine now.